### ISSUE TITLE
Release Native Plugins for 1.1.0 on S3 Production Bucket

### ISSUE THREAD

**Issue by peterzhuamazon** [opensearch-project/opensearch-build#686]

**Comment by peterzhuamazon** [comment]
```
$ ./opensearch-plugin install repository-s3
-> Installing repository-s3
-> Downloading repository-s3 from opensearch
[=================================================] 100%
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@     WARNING: plugin requires additional permissions     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
* java.lang.RuntimePermission accessDeclaredMembers
* java.lang.RuntimePermission getClassLoader
* java.lang.reflect.ReflectPermission suppressAccessChecks
* java.net.SocketPermission * connect,resolve
* java.util.PropertyPermission opensearch.allow_insecure_settings read,write
See http://docs.oracle.com/javase/8/docs/technotes/guides/security/permissions.html
for descriptions of what these permissions allow and the associated risks.

Continue with installation? [y/N]y
-> Installed repository-s3 with folder name repository-s3

$ ./opensearch-plugin install analysis-icu
-> Installing analysis-icu
-> Downloading analysis-icu from opensearch
[=================================================] 100%
-> Installed analysis-icu with folder name analysis-icu

$ ll ../plugins | grep icu
drwxr-xr-x 2 opensearch opensearch 4096 Oct  5 20:57 analysis-icu

$ ll ../plugins | grep s3
drwxr-xr-x 2 opensearch opensearch 4096 Oct  5 20:57 repository-s3
```

**Comment by peterzhuamazon** [comment]
We have released the 1.1.0 native plugins to releases channel.
